vignettes/advanced-use.Rmd

```diff
@@ -0,0 +1,58 @@
+---
+title: "Advanced Uses"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{advanced-use}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(npi)
+library(purrr)
+```
+
+This vignette provides an brief exploration to advanced uses of the npi package.
```

Comment:
"This vignette explores advanced uses of the npi package."

Reply:
Done.